src/eureka/S4_generate_lightcurves/s4_genLC.py

```diff
@@ -525,9 +525,10 @@ def genlc(eventlabel, ecf_path=None, s3_meta=None, input_meta=None):
                 if meta.isplots_S4 >= 3:
                     plots_s4.binned_lightcurve(meta, log, lc, 0, white=True)
 
-            # Generate limb-darkening coefficients
-            if hasattr(meta, 'compute_ld') and meta.compute_ld:
-                log.writelog("Generating limb-darkening coefficients...",
+            # Generate ExoTiC limb-darkening coefficients
+            if (meta.compute_ld == 'exotic-ld') or \
```

Comment:
Although we don't actually have a check, this isn't the first reference to `meta.compute_ld`.  It has existed in the S4 ECF since nearly the beginning, so I'm not worried about backwards compatibility.